.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/jcsda/spack
-  branch = release/1.5.1
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/jcsda/spack
+  #branch = release/1.5.1
+  url = https://github.com/climbfuji/spack
```

Comment:
Do these need to be reverted now that https://github.com/JCSDA/spack/pull/341 is merged? Also update the spack submodule hash?

Reply:
done